app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +27,26 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          scope.storageClasses = _.map(storageClasses.by('metadata.name'));
+          var defaultClassAnnotation = "storageclass.beta.kubernetes.io/is-default-class";
+          _.each(scope.storageClasses, function(sclass){
+            if ("annotations" in sclass.metadata && defaultClassAnnotation in sclass.metadata.annotations && sclass.metadata.annotations[defaultClassAnnotation] === "true") {
+              scope.defaultStorageClass = sclass.metadata.name;
+            }
+          });
+          if (scope.defaultStorageClass === "")  { //if there is no default, set a no storage class option
+            var noclass = {
+              metadata: {
+                name: "No Storage Class",
+                labels: {},
+                annotations: {
+                  description: "No storage class will be assigned unless a default class has been assigned by the system administrator."
```

Comment:
This description doesn't make sense if we're only showing the option when there's no default.